libparsec/crates/platform_mountpoint/src/lib.rs

```diff
@@ -1,27 +1,55 @@
 // Parsec Cloud (https://parsec.cloud) Copyright (c) BUSL-1.1 2016-present Scille SAS
 
-// TODO: UNIX implementation is linux-only for the moment
 #[cfg(target_os = "linux")]
 mod unix;
-// Windows is mocked, so we can use it for macOS too for the moment...
-#[cfg(any(target_os = "windows", target_os = "macos"))]
+#[cfg(target_os = "windows")]
 mod windows;
 
 #[cfg(target_os = "linux")]
 pub(crate) use unix as platform;
-#[cfg(any(target_os = "windows", target_os = "macos"))]
+#[cfg(target_os = "windows")]
 pub(crate) use windows as platform;
 
+// TODO: UNIX implementation is linux-only for the moment
```

Comment:
Update comment

Reply:
This comment is still relevant: macOS is not supported for the FUSE implementation (cf. the `use std::{os::linux::fs::MetadataExt, sync::Arc, thread::JoinHandle};` in `mount.rs`)